### ISSUE TITLE
A11y_.NET Core_ASP.NETCore_AngularForgotPassword_ScreenReader: No asterisk marks are provided for the required fields

### ISSUE THREAD

**Issue by Truptik12** [dotnet/aspnetcore#42337]
[42337_A11y_.NET Core_ASP.NETCore_AngularForgotPassword_ScreenReader.zip](https://github.com/dotnet/aspnetcore/files/8973835/42337_A11y_.NET.Core_ASP.NETCore_AngularForgotPassword_ScreenReader.zip)

“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags**:#A11yMAS;#A11yTCS;#.NET Core[aspnetcore]-Win32-June2022;#.NET Core;#A11ySev2;#WCAG3.3.2;#DesktopWeb;#Win11;#Benchmark;#NVDA;#Narrator;

### Environment Details:
.NET Core
ASP.NET Core
Operating System: Windows 11 Enterprise 21H2
Chrome: Version 102.0.5005.115 (Official Build) (64-bit)

### Steps to Reproduce:

1. Open command prompt.
2. Create a folder using command "mkdir foldername" e.g. "mkdir Angular"
3. Move to the created directory using command "cd foldername"
4. Run the following command "dotnet new angular -au Individual" and hit enter.
5. Run the following command "dotnet run"
6. Copy the First URL link from Command prompt and copy it
7. Paste it in URL section of Browser and hit Search
8. Browser page will open with above template loaded
9. Hit enter on login link
10. Tab and navigate to "forgot your password?" link and press enter.
11. The screen reader announces "Email" is a required field, but no asterisk is provided.

### Actual:
No asterisk marks are provided for the required fields.

**This issue is observed throughout the application for all the required fields.**
**This is also observed with chromium edge and Narrator**

### Expected:
Asterisk marks should be provided for the required fields.

### User Impact:
Low vision users face difficulty to identify which control is required.

**Comment by HaoK** [comment]
Fix is probably adding some css to required form fields by default:

```
.form-group.required .control-label:after {  content:"*";  color:red;}
```

**Comment by HaoK** [comment]
@merriemcgaw for 6.0, we added area-required for our forms, so for example for login username, we have area-required=true, is that sufficient indication for which inputs are required?

```
<input class="form-control" autocomplete="username" aria-required="true" type="email" 
```

**Comment by merriemcgaw** [comment]
Your comments make a lot of sense to me. Let's set up some time in Office Hours to discuss this issue. I think we might be able to get them to lower it to Sev3 which we don't have an SLA around.

**Comment by HaoK** [comment]
Per discussion in accessibility office hours, the current product behavior is sufficient and this bug should be closed, reassigning back to the OP

**Comment by Truptik12** [comment]
Closing the bug as per the above comments.
